template.yaml

```diff
@@ -22,7 +22,7 @@ Parameters:
   DataCiteMdsConfigs:
     Type: String
     Description: Institution configs for DataCite MDS API
-    Default: '{{resolve:secretsmanager:dataCiteMdsConfigs:SecretString}}'
+    Default: '{{resolve:secretsmanager:dataciteMdsConfigs:SecretString}}'
```

Comment:
@janbj :  We need to create a new one, as config has changed to be more descriptive as well. instead of institution id , it is customerId  for example.

Reply:
Where can I find what the complete new config is supposed to be?

Reply:
We probably should create a task for setting this up for our customers in test and production. 
Described the configuration here: https://github.com/BIBSYSDEV/nva-doi-registrar-client/pull/30

Reply:
Added `dataCiteCustomerSecrets` , `develop` README is updated.

A task for dynamically/tool updating the secret manager with updated customer configuration for new customers is created after first new customer user is logging in, is something that needs to be looked into.

Outside the scope of this PR, but needs to be handled. @brinxmat